doc/user/content/overview/open-cdc.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Materialize Open CDC"
+description: "The Open CDC format allows Materialize to detect when records are duplicated or out of order."
+menu:
+  main:
+    parent: "overview"
+    weight: 4
+---
+
+
+Change data capture (CDC) tools provide feeds that record any changes to a database. Typically, the feeds are then saved to another platform, like Kafka, for storage or processing.
+
+However, sometimes the stream can have missing or duplicate records, or send records out of order. For example, if Debezium crashes while writing a record, it may retry and write the record again, resulting in a duplicate entry.
+
+To prevent the introduction of this kind of error, Materialize has developed the **Open CDC format**. The Open CDC format carries enough information for a downstream data consumer (like Materialize) to recognize when records are missing, duplicated, or out of order. Each record contains:
+
+- the fields being changed
+- the timestamp for the change
```

Comment:
@umanwizard ^ see above

Reply:
logical timestamp, i.e., whatever the user wants it to mean.

I think we can punt on fully explaining this for now; we probably at some point need to write up what "timestamps" mean in the context of Materialize.

Reply:
Just for context, Micole, a logical timestamp is something used in distributed systems to identify the ordering of some set of events; a physical time typically represents some amount of time from the Unix epoch. These two domains are very different and if one assumes that they have one when they have the other, they're going to be in for a really bad time. This is an important distinction to make that the timestamp field we expect here is a logical timestamp.

Reply:
it's a bit confusing because our logical timestamps usually are taken from wall-clock time, but... yeah.

The important thing is that timestamps represent transaction boundaries, for us: i.e., if two things happen at the same timestamp, they will be observed atomically.